stl/inc/xmemory

```diff
@@ -782,6 +782,8 @@ class allocator {
 public:
     static_assert(!is_const_v<_Ty>, "The C++ Standard forbids containers of const elements "
                                     "because allocator<const T> is ill-formed.");
+    static_assert(!is_reference_v<_Ty>, "The C++ Standard forbids allocators for reference elements "

```

Comment:
I would phrase both messages the same

Reply:
I see your point and agree

Reply:
These are different because the original `is_const_v` check is meant to be seen while instantiating containers, while the new check has dedicated occurrences for containers, so this one will be seen by itself only when the allocator is being independently used. I think that's sufficient reason for slightly different wording.